### DISCUSSION TITLE
cgitb deprecation python 3.13

### DISCUSSION THREAD

**Discussion by ginolegigot** [apache/airflow#41752] [Q&A]
Hello!
I ran across this warning today after installing airflow 2.10.0 in python 3.12. Is It a way to avoid this warning to be displayed without changing airflow code ? Is this warning OK ?
```bash
~# airflow users list
/airflow/lib/python3.12/site-packages/airflow/plugins_manager.py:30 DeprecationWarning: 'cgitb' is deprecated and slated for removal in Python 3.13
```

**Comment by marcosbc** [comment]
Fixed in 2.11.0 - #41793
